package.json

```diff
@@ -20,14 +20,14 @@
   },
   "scripts": {
     "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
-    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test": "mocha --recursive --reporter spec --timeout 3000 --ui bdd --exit",
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
     "lint:md": "remark 'README.md' documentation",
     "lint:text": "textlint 'README.md' 'documentation/*.md' 'documentation/**/*.md'",
     "prettier": "prettier --config .prettierrc.json --write ./**/**/*.js **/*.js *.js",
     "prettier:text": "prettier 'README.md' 'documentation/*.md' 'documentation/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
-    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
```

Comment:
Why this change?

Reply:
This is a mistake in the merge, sorry.

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad